AutomaticInterface/AutomaticInterface/Builder.cs

```diff
@@ -18,6 +18,15 @@ public static class Builder
                 | SymbolDisplayParameterOptions.IncludeParamsRefOut
         );
 
+    private static readonly SymbolDisplayFormat TypeDisplayFormat =
```

Comment:
New display format for types. Based on `SymbolDisplayFormat.FullyQualifiedFormat`, but with support for nullables, and without unnecessary non-type parameters